pkgs/by-name/be/bend/package.nix

```
@@ -0,0 +1,45 @@
+{
+  rustPlatform,
+  fetchCrate,
+  fetchFromGitHub,
+  lib,
+}:
+
+let
+  hvm = rustPlatform.buildRustPackage rec {
+    pname = "hvm";
+    version = "2.0.12";
+    src = fetchCrate {
+      inherit pname version;
+      hash = "sha256-/55SK/5zBKXmucRQPoYt/8IHxisQlOxNEVMAZVMtCNI=";
+    };
+    RUSTC_BOOTSTRAP = true;
+    cargoHash = "sha256-9U8Y0KaQHIfOZnCKbl94VvjS/7Qmi6UnKMDZDTXcye0=";
+  };
+in
+rustPlatform.buildRustPackage rec {
+  pname = "Bend";
+  version = "0.2.9";
+
+  src = fetchFromGitHub {
+    owner = "HigherOrderCO";
+    repo = "Bend";
+    rev = "refs/tags/${version}";
+    hash = "sha256-MEfB2SBJN7uEwfZGoEL7DQXsr1fccdZyGyzHtNv9wow=";
+  };
+
+  cargoHash = "sha256-+i+Y3MgCBVN3REmPwAjm2SiF9FJ0i05czmPKB8JtAFM=";
+
+  RUSTC_BOOTSTRAP = true;
+
+  nativeBuildInputs = [ hvm ];
```

```suggestion
  nativeBuildInputs = [ hvm makeWrapper ];
  postInstall = ''
    wrapProgram $out/bin/bend \
      --prefix PATH : ${lib.makeBinPath [ hvm ]}
  '';
```

Bend requires hvm at runtime. This is one way to make it available, as suggested [here](https://github.com/NixOS/nixpkgs/issues/312527#issuecomment-2118508913).

Without this, trying to run bend against a file results in this error:

```
bend run sum.bend
Errors:
While running hvm: No such file or directory (os error 2)
```